rfcs/0009-enclave.md

```diff
@@ -31,6 +31,15 @@ Secure Enclaves (such as SGX) provide a layer of memory and computation protecti
     * The sign request contains the entry ID for the new vote
     * Enclave verifies that new vote's entry ID is on the correct branch (following the rules #1 and #2 above)
 
+
+## Branch determination
+
+Due to the fact that the enclave cannot process PoH, it has no direct knowledge of branch history of a submitted validator vote. Each enclave should be initiated with the current `active set` of public keys. A validator should submit it's current vote along with the votes of the active set (including itself) that it observed in the slot of it's previous vote. In this way, the enclave can surmise the votes accompanying the validator's previous vote and thus the branch being voted on. This is not possible for the validators initial submitted vote, as it will not have a 'previous' slot to reference, so a short voting freeze should apply until the second vote is submitted containing the votes within the active set, along with it's own vote, at the height of the initial vote.
```

Comment:
"it's" should be "its" in two places. `active set` should use asterisks to make it italic instead of fixed width. Folks typically use those backticks in markdown to represent identifiers from code. "validators" should be "validator's". "so a short voting freeze ..." makes that sentence a run-on.

Reply:
addressed, thanks!